astropy/nddata/nduncertainty.py

```diff
@@ -144,6 +144,13 @@ def unit(self):
                 return self.parent_nddata.unit
         return self._unit
 
+    @property
+    def quantity(self):
+        """
+        This uncertainty as an `~astropy.units.Quantity` object.
+        """
+        return u.Quantity(self.array, self.unit, copy=False)
```

Comment:
Obviously I didn't do a thorough enough review, but CI is failing because we didn't do `from astropy import units as u`, we imported `Quantity`. Change that and I think the tests will pass.